twitter-borrowbot/makefile

```diff
@@ -0,0 +1,2 @@
+go:
+	pm2 start twitterbot.py
```

Comment:
This allows folks to run the bot on a server using pm2 https://pm2.keymetrics.io/

Reply:
No problem in this case, though this seems like a great candidate for a separate PR